### PR TITLE
Hide delete button from not authenticated users

### LINKED ISSUE #297: Don't show "bin" icon in the cards when not authenticated
As for now a "bin/trash can" icon appears in the cards of associated documents (for instance in a WP page). It should only appear for users with permission "moderator".

### PR DESCRIPTION
**Pull request by ginold**
fixes #297

### CODE REVIEW

File: c2corg_ui/static/js/documentediting.js [hunk 1]
```diff
@@ -171,6 +171,8 @@ app.DocumentEditingController = function($scope, $element, $attrs,
   if (this.auth_.isAuthenticated()) {
 
     if (this.id_ && this.lang_) {
+      // check each n seconds if user has logged out in between
+      this.recheckAuth_();
```
Comment (asaunier) [inline c2corg_ui/static/js/documentediting.js]: Why not doing it also when creating a new document?

And honestly I would be rather pissed off if I was redirected  to the auth page while editing a doc (losing my work?).

**Comment by asaunier** [comment]
What the connection of commit "redirect user that has logged out during editing" with the association removal button? Please create a separated PR.

Anyway as commented above, I think this change is not user friendly as is => I would remove the commit from this PR.

**Comment by ginold** [comment]
OK, i'll move it. 
It will not be lost because it will be saved to the localstorage. And what's the point if i disconnect myself and then have to login back in order to save the document, because i have no rights? it would be lost in the same way.

**Comment by asaunier** [comment]
As for now there is no localstorage saving.
It would be better to add your change when the localstorage saving is done (in the same PR?). 
Actually I think it is no use to redirect the user to the auth page while he is editing: we'd better check the user is still authenticated when saving the editing form (we'll have to do it anyway to make sure the changes are not lost), so no need to do it before.

**Comment by ginold** [comment]
I moved the commit to a different branch/pr. 
Besides, ok for merging?

**Comment by asaunier** [comment]
Thanks, please merge.

**Pull request merged**